gdc_client/upload/client.py

```diff
@@ -65,7 +61,7 @@ def __getattr__(self, attr):
         return getattr(self._file, attr)
 
     def read(self, num):
-        self.pbar.update(min(self.pbar.value + num, self.filesize))
+        self.pbar.update(num)
```

Comment:
is the min filesize logic no longer applicable?

Reply:
we're not calculating the actual value of the progress bar as we did before, but I guess updating with an actual size that's been read would be better here